contracts/GenArt721CoreV3.sol

```diff
@@ -667,4 +652,21 @@ contract GenArt721CoreV3 is ERC721, IGenArt721CoreContractV3 {
                 )
             );
     }
+
+    function owner()
+        public
+        view
+        override(Ownable, IGenArt721CoreContractV3)
+        returns (address)
+    {
+        return Ownable.owner();
+    }
```

Comment:
Nit: can we add a documentation comment here that clarifies that by inheriting Ownable, the owner is set to the deployer by default on deployment and link to this: https://docs.openzeppelin.com/contracts/2.x/api/ownership#Ownable ? 

Alternatively may make more sense to add this note in the initializer; WDYT?

Reply:
good catch - added docstrings to this function, noted deployment behavior, and linked to OpenZeppelin docs for v4 ownable!

Reply:
TYVM!! :)